next.config.mjs

```diff
@@ -10,6 +10,7 @@ const withMDX = MDX();
 /** @type {import('next').NextConfig} */
 const nextConfig = {
 	reactStrictMode: true,
+	output: "standalone",
```

Comment:
I think docker is optional, should we add standalone by default?  
My proposal **commented by default** with an appropriate comment. 
Like below or similar
```
const nextConfig = {
	reactStrictMode: true,
        // uncomment the following line in case you build to docker
        // output: "standalone",
```

Reply:
That's a very good idea